apps/server/src/modules/task/controller/task-copy.controller.ts

```diff
@@ -0,0 +1,31 @@
+import { Body, Controller, Param, Post } from '@nestjs/common';
+import { ApiTags } from '@nestjs/swagger';
+import { ParseObjectIdPipe } from '@shared/controller/pipe';
+import { ICurrentUser } from '@shared/domain';
+import { Authenticate, CurrentUser } from '@src/modules/authentication/decorator/auth.decorator';
+import { TaskCopyMapper } from '../mapper/task-copy.mapper';
+import { TaskCopyUC } from '../uc/task-copy.uc';
+import { TaskCopyApiParams } from './dto/task-copy.params';
+import { TaskCopyApiResponse } from './dto/task-copy.response';
+
+@ApiTags('TaskCopy')
```

Comment:
So there are a couple things here.
First, we forgot to add this controller, the uc, and the copyservice to the module (2/3rds of that is my fault). After doing that, the api looks like this:

![grafik](https://user-images.githubusercontent.com/10486254/168559537-87304513-0b30-42fd-9b89-52cc05b52997.png)

The tag controlls where in the docu the route shows up. I think, it would be less confusing to add the copy route into the tasks section itself, by using the apitag 'Task'.

Reply:
after fixing this and the point below, you will notice that your route can either be added first, or last. 
![grafik](https://user-images.githubusercontent.com/10486254/168561059-ab745efb-3ef2-4a61-b697-48f93f830549.png)

Both looks kinda wierd... So I would suggest for simplicity to simply add your new controller function into the existing one for tasks.